gridgencore.py

```diff
@@ -1,8 +1,10 @@
 # This file is part of Infinity Grid Generator, view the README.md at https://github.com/mcmonkeyprojects/sd-infinity-grid-generator-script for more information.
 
-import os, glob, yaml, json, shutil, math, re, time
-from copy import copy
+import os, glob, yaml, json, shutil, math, re, time, types, threading
+from copy import copy, deepcopy
 from PIL import Image
+from modules.processing import StableDiffusionProcessing, Processed, StableDiffusionProcessingImg2Img, StableDiffusionProcessingTxt2Img
```

Comment:
gridgencore is not exclusive to Auto WebUI, this import cannot be here

Reply:
This is a generic grid generator script than can work in any python-backed environment, any auto-webui-specific impls need to be in the auto-webui file